doc/sphinx/practical-tools/coq-commands.rst

```diff
@@ -190,13 +177,10 @@ and ``coqtop``, unless stated otherwise:
   NTFS (Windows) or HFS+ (MacOS X) file systems is on the contrary to
   disallow two files differing only in the case in the same directory.
 
-  .. seealso:: Section :ref:`names-of-libraries`.
-:-R *directory dirpath*: Do as ``-Q`` *directory dirpath* but make the
-  subdirectory structure of *directory* recursively visible so that the
-  recursive contents of physical *directory* is available from Coq using
-  short or partially qualified names.
+  .. seealso:: Section :ref:`names-of-libraries`, :ref:`libraries-and-filesystem` and :cmd:`Require`.
+:-R *directory dirpath*: Do as ``-Q`` *directory dirpath* but allow using :cmd:`Require` with an implicit prefix.
```

Comment:
```suggestion
:-R *directory dirpath*: Similar to ``-Q`` *directory dirpath*, but allow using
  :cmd:`Require` with a partially qualified name.
```

"Do as ..." is odd wording
We don't use the term "implicit prefix" in the doc.  We do use "partially qualified identifier" (defined in Core Language/Modules/Qualified Identifiers).